packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/tasks/ImagePickerResultTask.kt

```diff
@@ -0,0 +1,16 @@
+package expo.modules.imagepicker.tasks
+
+import android.content.ContentResolver
+import android.net.Uri
+import android.os.AsyncTask
+import expo.modules.imagepicker.fileproviders.FileProvider
+import org.unimodules.core.Promise
+import java.io.File
+
+/**
+ * Base class for tasks which will process result of other activities like system camera or crop tool.
+ */
+abstract class ImagePickerResultTask(protected val promise: Promise,
```

Comment:
Do we need this class? If it does not have any methods, what's the point? We have to repeat all constructor entries in every subclass anyway.